src/main/java/com/massivecraft/factions/tax/format/TimeUnit.java

```diff
@@ -0,0 +1,175 @@
+package com.massivecraft.factions.tax.format;

```

Comment:
Entire class is tabs instead of spaces.

Also why is a utility class buried inside a tax package?